api/app/src/main/kotlin/packit/service/OutpackServerClient.kt

```diff
@@ -1,30 +1,64 @@
 package packit.service
 
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.apache.tomcat.util.http.fileupload.IOUtils
 import org.slf4j.LoggerFactory
 import org.springframework.core.ParameterizedTypeReference
 import org.springframework.http.*
+import org.springframework.http.client.ClientHttpRequest
 import org.springframework.stereotype.Service
+import org.springframework.web.client.HttpStatusCodeException
 import org.springframework.web.client.RestTemplate
 import packit.AppConfig
 import packit.exceptions.PackitException
 import packit.model.OutpackMetadata
 import packit.model.OutpackResponse
 import packit.model.PacketMetadata
+import java.net.URI
 
-interface OutpackServer {
+interface OutpackServer
+{
     fun getMetadata(from: Long? = null): List<OutpackMetadata>
-    fun <T> get(urlFragment: String): T
     fun getMetadataById(id: String): PacketMetadata?
     fun getFileByHash(hash: String): Pair<ByteArray, HttpHeaders>?
+    fun proxyRequest(urlFragment: String, request: HttpServletRequest, response: HttpServletResponse)
+    fun getChecksum(): String
 }
 
 @Service
 class OutpackServerClient(appConfig: AppConfig) : OutpackServer
 {
-    private val baseUrl: String = appConfig.outpackServerUrl
+    val baseUrl: String = appConfig.outpackServerUrl
 
     private val restTemplate = RestTemplate()
 
+    override fun proxyRequest(urlFragment: String, request: HttpServletRequest, response: HttpServletResponse)
+    {
+        val url = "$baseUrl/$urlFragment"
+        val method = request.method
+        log.debug("{} {}", method, url)
```

Comment:
Should remove this debug log?

Reply:
This is included deliberately on all methods in this class.